pkg/bff/server.go

```diff
@@ -121,7 +109,27 @@ func New(conf config.Config) (s *Server, err error) {
 	return s, nil
 }
 
-type NetworkClient struct {
+// ConnectNetwork creates a unified client to the TRISA Directory Service and TRISA
+// members service specified in the configuration. This method is used to connect to
+// both the TestNet and the MainNet so we can maintain separate clients for each.
+func ConnectNetwork(conf config.DirectoryConfig) (client *GDSClient, err error) {
```

Comment:
Or if you prefer instead of naming the interface `GlobalDirectoryClient` we could keep the name `NetworkClient` and have the implementation be the `GDSClient`.